x/axelarnet/client/cli/tx.go

```diff
@@ -345,7 +345,26 @@ func getCmdCallContract() *cobra.Command {
 				return err
 			}
 
-			msg := types.NewCallContractRequest(clientCtx.GetFromAddress(), args[0], args[1], payload)
+			var fee *types.Fee = nil
```

Comment:
fee needs to be optional. Check out https://github.com/axelarnetwork/axelar-core/blob/5b8d0f32dc894f9bb8330ce5016c8d9efd098c49/x/evm/client/cli/tx.go#L249.